src/main/java/org/jenkinsci/plugins/pipeline/github/trigger/IssueCommentCause.java

```diff
@@ -9,9 +10,28 @@
  */
 public class IssueCommentCause extends Cause {
     private final String userLogin;
+    private final String comment;
+    private final String triggerPattern;
 
-    public IssueCommentCause(final String userLogin) {
+    public IssueCommentCause(final String userLogin, final String comment, final String triggerPattern) {
         this.userLogin = userLogin;
+        this.comment = comment;
+        this.triggerPattern = triggerPattern;
+    }
+
+    @Whitelisted
+    public String getUserLogin() {
+        return userLogin;
+    }
+
+    @Whitelisted
+    public String getComment() {
+        return comment;
+    }
+
+    @Whitelisted
+    public String getTriggerPattern() {
+        return triggerPattern;
     }
 
     @Override
```

Comment:
Should we update the `getShortDescription()` method to include the comment and trigger patten? 

It won't really be "short" anymore but at least it'll be descriptive and specific.